### ISSUE TITLE
Not enough values error in tiktoken.get_encoding

### ISSUE THREAD

**Issue by Ayan-Bandyopadhyay** [openai/tiktoken#114]
the line`encoding = tiktoken.get_encoding("cl100k_base")` gives me the following error:

```
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.10/site-packages/tiktoken/registry.py", line 63, in get_encoding
    enc = Encoding(**constructor())
  File "/usr/local/lib/python3.10/site-packages/tiktoken_ext/openai_public.py", line 64, in cl100k_base
    mergeable_ranks = load_tiktoken_bpe(
  File "/usr/local/lib/python3.10/site-packages/tiktoken/load.py", line 115, in load_tiktoken_bpe
    return {
  File "/usr/local/lib/python3.10/site-packages/tiktoken/load.py", line 117, in <dictcomp>
    for token, rank in (line.split() for line in contents.splitlines() if line)
ValueError: not enough values to unpack (expected 2, got 1)
```

This is on tiktoken version 0.3.3

**Comment by gaojing8500** [comment]
comment list code
![image](https://user-images.githubusercontent.com/20908110/234242881-41eae797-f426-48c2-8ce1-5565ebb4ad60.png)

**Comment by hauntsaninja** [comment]
I've released tiktoken 0.4 which has better error handling for cases like this. If a bad file has gotten cached, gaojing8500's suggestion works as does setting the environment variable `TIKTOKEN_CACHE_DIR=''`

**Comment by Miuler** [comment]
A question, why the mergeable_ranks is downloaded in runtime? why not to have it downloaded in the repo?

```python
def gpt2():
    mergeable_ranks = data_gym_to_mergeable_bpe_ranks(
        vocab_bpe_file="https://openaipublic.blob.core.windows.net/gpt-2/encodings/main/vocab.bpe",
        encoder_json_file="https://openaipublic.blob.core.windows.net/gpt-2/encodings/main/encoder.json",
    )
    return {
        "name": "gpt2",
        "explicit_n_vocab": 50257,
        "pat_str": r"""'s|'t|'re|'ve|'m|'ll|'d| ?\p{L}+| ?\p{N}+| ?[^\s\p{L}\p{N}]+|\s+(?!\S)|\s+""",
        "mergeable_ranks": mergeable_ranks,
        "special_tokens": {"<|endoftext|>": 50256},
    }
```
